<launch>
  <param name="self_ip" type="str" value="195.0.0.60" />
  <param name="robot_ip" type="str" value="195.0.0.20" />
  <param name="udp" type="bool" value="true" />
  <param name="udp_port" type="int" value="50008" />
  <node pkg="external_interface" type="vehicle_output.py" name="send_vehicle_output" respawn="true" output="screen" />
  <node pkg="external_interface" type="forwarding_rtk_correction.py" name="forward_rtk" respawn="true" output="screen" />
  <!-- <node pkg="external_interface" type="VehBrakeGPIO.py" name="vehicle_brake_gpio" respawn="true" output="screen" /> -->
</launch>